sdk/core/azure-mgmt-core/azure/mgmt/core/_async_pipeline_client.py

```diff
@@ -47,23 +42,11 @@ def __init__(self, base_url, **kwargs):
                 raise ValueError(
                     "Current implementation requires to pass 'config' if you don't pass 'policies'"
                 )
-            kwargs["policies"] = self._default_policies(**kwargs)
+            per_call_policies = kwargs.get('per_call_policies', [])
+            per_call_policies.append(AsyncARMAutoResourceProviderRegistrationPolicy())
+            kwargs["per_call_policies"] = per_call_policies
+            config = kwargs.get('config')
+            if not config.http_logging_policy:
+                config.http_logging_policy = kwargs.get('http_logging_policy', ARMHttpLoggingPolicy(**kwargs))
```

Comment:
These lines are unnecessary as the client will always have `http_logging_policy` configured:

https://github.com/Azure/azure-sdk-for-python/blob/28545a0bfd5586dcf2de8162cd4af9bfaa72f7f6/sdk/resources/azure-mgmt-resource/azure/mgmt/resource/subscriptions/_configuration.py#L59

Reply:
We need it because in your test case, you use the azure.core.configure which does not honor http_logging_policy.

w/o it will fail the tests.